pkg/reconciler/sinkbinding/controller.go

```diff
@@ -144,7 +143,7 @@ func NewController(
 	}
 
 	c.WithContext = func(ctx context.Context, b psbinding.Bindable) (context.Context, error) {
-		return v1.WithTrustBundleConfigMapLister(v1.WithURIResolver(ctx, sbResolver), trustBundleConfigMapLister), nil
+		return v1.WithTrustBundleConfigMapLister(v1.WithURIResolver(ctx, sbResolver), trustBundleConfigMapInformer.Lister()), nil
```

Comment:
@creydr what's the reason for this change?

Reply:
Never mind, I missed the fact that there were two different configmap informers happening here